client/web/compose/src/components/ModuleFields/Configurator/validation.vue

```diff
@@ -116,6 +116,78 @@
         {{ $t('validators.description') }}
       </b-form-text>
     </b-form-group>
+
+    <div>
+      <div>
+        <h5>{{ $t('constraints.title') }}</h5>
+        <b-form-checkbox
+          v-model="fieldConstraintExists"
+          class="mt-3"
+          @change="toggleFieldConstraint"
+        >
+          {{ $t('constraints.description') }}
+        </b-form-checkbox>
+      </div>
+
+      <div class="mt-4">
+        <b-row>
+          <b-col
+            cols="12"
+            sm="6"
+          >
+            <b-form-group
+              :label="$t('constraints.valueModifiers')"
+            >
+              <b-form-select
+                v-model="constraint.modifier"
+                :options="modifierOptions"
+              >
+                <template #first>
+                  <b-form-select-option
+                    :value="undefined"
```

Comment:
Ask vivek if this '' value actually means the modifier is "case sensitive"?
If so then None should not exist and case-sensitive should be default

Reply:
response is NO

Reply:
<img width="1272" alt="deDup_updates" src="https://user-images.githubusercontent.com/6803329/212282842-25c0076d-1db9-43fa-ad6c-4e51b1565a70.png">

Reply:
```
{
  name: "case-sensitive",
  strict: true,
  constraints: [
    {
      "attribute": "field-foo",
      "modifier": "ignore-case",
      "multiValue": "equal"
    },
    {
      "attribute": "field-bar",
      "modifier": "fuzzy-match",
      "multiValue": "one-of"
    }
  ]
}
```

Reply:
As agreed the name will be empty now by default, and a new modifier called `case-sensitive` will be added